library/ecp.c

```diff
@@ -1338,6 +1340,9 @@ static int ecp_normalize_jac( const mbedtls_ecp_group *grp, mbedtls_ecp_point *p
     mbedtls_mpi_free( &Zi ); mbedtls_mpi_free( &ZZi );
 
     return( ret );
+#else /* !defined(MBEDTLS_ECP_NO_FALLBACK) || !defined(MBEDTLS_ECP_NORMALIZE_JAC_ALT) */
+    return( MBEDTLS_ERR_ECP_FEATURE_UNAVAILABLE );
```

Comment:
I'm not sure to follow. In any of the four possible combinations of the two config flags, the function does something. When MBEDTLS_ECP_NORMALIZE_JAC_ALT and MBEDTLS_ECP_NO_FALLBACK are defined together, the function just calls mbedtls_internal_ecp_normalize_jac() if necessary but is not empty.

Reply:
I'm with Ronald on this one. The line you are referring to with this comment is only there when both `MBEDTLS_ECP_NORMALIZE_JAC_ALT` and `MBEDTLS_ECP_NO_FALLBACK` are defined. And in that case, because `MBEDTLS_ECP_NORMALIZE_JAC_ALT` is defined, the function will have done something: tried the operation through the ALT driver if the driver supports it.

Reply:
Right, I get it, sorry.